pages/css/documentation.md

```diff
@@ -4,15 +4,33 @@ permalink: /css/documentation/
 sidenav: css
 ---
 # Documentation
+
+You should always document:
+- Magic numbers (when you absolutely cannot avoid using them)
+- Overrides of imported frameworks like USWDS
+- Rules that must be used in conjunction with one another
+(using BEM naming conventions is somewhat self-documenting in this regard)
```

Comment:
I'd welcome any additions to this list!

Reply:
Top level components you create, to leave a note of what they are, how they're used, any a11y concerns, etc.